guides/release-notes.md

```diff
@@ -7,6 +7,25 @@ We love hearing your feedback! You can help us improve the docs by reporting any
 ![Was this helpful, yes/no buttons](../media/feedback.png)
 
 ---
+### New API: Dashboard Elements
+
+`Wix Editor Elements ($w)`
+
+The new [Dashboard Elements](https://www.wix.com/velo/reference/$w/dashboardbutton) API allows you to create customized, interactive dashboards for your users.
```

Comment:
> `[Dashboard Elements](https://www.wix.com/velo/reference/$w/dashboardbutton)`

What you describe in the link text and what you actually link to need to be the same thing. The link text gives a hint of what the user can expect when they click; best not to surprise them with something else.